v7/zen-jinja/conf.py.sample

```diff
@@ -1,11 +1,11 @@
 NAVIGATION_LINKS = {
     DEFAULT_LANG: (
-        ('/index.html', 'Home', 'icon-home'),
-        ('/archive.html', 'Archives', 'icon-folder-open-alt'),
-        ('/categories/index.html', 'Tags', 'icon-tags'),
-        ('/rss.xml', 'RSS', 'icon-rss'),
-        ('https://getnikola.com', 'About me', 'icon-user'),
-        ('https://twitter.com/getnikola', 'My Twitter', 'icon-twitter'),
-        ('https://github.com/getnikola', 'My Github', 'icon-github'),
+        ('/index.html', 'Home', 'fa-home'),
```

Comment:
You need the `fa` prefix as well (eg. `fa fa-home`).

Reply:
I'd prefer to add the prefix to the `arusahni_helper.tmpl` to avoid adding it to every navigation item:
```html
<%def name="html_navigation_links()">
[...]
    <li><a href="${url}" title="${text}"><i class="fa ${icon} fa-3x"></i></a></li>
[...]
```
Is this a valid option?

Reply:
That can work too, although (a) why `fa-3x` if we don’t need it now? (b) that wouldn’t cut it for font awesome v5 if someone wanted to use it

Reply:
a) an attempt of making use of an existing class element (`fa-Xx` sizes) instead of 'font-size' declaration in the css...(you started with getting rid of cruft ;))

b) thanks for clarification, I haven't gone in for FA5.

Reply:
`font-size` would make it a bit more flexible

Reply:
Setting `font-size` in CSS is more flexible than static `.fa-2x/3x` sizing.